Bug Fix:  Thanks to https://github.com/YKdvd  for pointing out that its not needed to convert // to /

louisVottero opened the pull request:
removed from util_file.fix_slashes()
```
if not directory.find('https://') > -1:
        directory = directory.replace('//', '/')
```

YKdvd commented:
We just deleted those same lines you did in fix_slashes(), and I it seems to have worked for them.  

I notice you also removed a couple of calls to fix_slashes(), as well.  Those calls didn't use the return value, and I don't think fix_slashes() would have modified its parameter in-place, so those were probably in effect do-nothing calls anyway?

Anyway, looks fine.

louisVottero commented:
> We just deleted those same lines you did in fix_slashes(), and I it seems to have worked for them.
> 
> I notice you also removed a couple of calls to fix_slashes(), as well. Those calls didn't use the return value, and I don't think fix_slashes() would have modified its parameter in-place, so those were probably in effect do-nothing calls anyway?
> 
> Anyway, looks fine.

Yep, I removed those calls because they weren't returning anything and were doing nothing.
